pandas/core/arraylike.py

```diff
@@ -97,6 +97,53 @@ def _arith_method(self, other, op):
 
     @unpack_zerodim_and_defer("__add__")
     def __add__(self, other):
+        """
+        Get addition of DataFrame and other, column-wise.
+
+        This is equivalent to df.add(other, axis=1).
+
+        Parameters
+        ----------
+        other : array-like
+            Object to be added to the DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            The result of adding ``other`` to DataFrame.
+
+        See Also
+        --------
+        DataFrame.add : Add a DataFrame and another object, with option for index-
+            or column-oriented addition.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({"A": [1, 2], "B": [2, 0]}, index=["AA", "BB"])
+        >>> df[["A", "B"]] + 1.5
+            A   B
+        AA  2.5 3.5
+        BB  3.5 1.5
```

Comment:
I see you used the example from the issue, but it'd be better to use more meaningful data, to make the examples easier and faster to understand. In the below example, when the DataFrame and Series are aligned, using `A` vs `AA` makes quite hard to see what's going on.

If we use a dataframe with let's say animals as we use in other places, it'd be pretty fast to see if the Series corresponds to a column or a row, and how it is aligned.